_security/configuration/disable-enable-security.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: default
+title: Disabling and enabling the Security plugin
+parent: Configuration
+nav_order: 40
+has_toc: true
+redirect_from: 
+ - /security-plugin/configuration/disable/
+---
+
+# Disabling and enabling the Security plugin
+
+The Security plugin is installed by default with OpenSearch, but you can disable it temporarily if you want to make testing or internal usage more straightforward. You can then enable it once you're ready to configure security for your cluster. 
+
+If you have your own security solution or need to remove the Security plugin for development purposes, you can uninstall the plugin completely. Note that OpenSearch Dashboards can run only against a secure cluster, so if you uninstall the OpenSearch Security plugin, you'll have to also uninstall the Dashboard plugin. 
+
+
+## Disabling security
+
+To disable the Security plugin for OpenSearch, add the following line in `opensearch.yml`:
+
+```yml
+plugins.security.disabled: true
+```
+
+## Removing the Security plugin
+
+If you want to remove the Security plugin in your OpenSearch instance without changing your configuration settings in `opensearch.yml`, use the following steps.
```

Comment:
```suggestion
While using the Security plugin is always recommended, if you do need to completely remove the Security plugin from your OpenSearch instance, you are able to still keep your other `opensearch.yml` settings. You can take the following steps to permanently remove the Security plugin from your cluster: 
```

Reply:
I'm updated to:
'''
While using the Security plugin is always recommended, if you do need to completely remove the Security plugin from your OpenSearch instance, you can take the following steps:
'''
As this is not necessary a permanent (irreversible) action, probably best to avoid "permanently"